src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -119,62 +119,52 @@ rpmostree_sysroot_upgrader_initable_init (GInitable        *initable,
                                           GCancellable     *cancellable,
                                           GError          **error)
 {
-  gboolean ret = FALSE;
   RpmOstreeSysrootUpgrader *self = (RpmOstreeSysrootUpgrader*)initable;
+
   OstreeDeployment *booted_deployment =
     ostree_sysroot_get_booted_deployment (self->sysroot);
-  const char *merge_deployment_csum = NULL;
-  gboolean is_layered;
-
   if (booted_deployment == NULL && self->osname == NULL)
-    {
-      g_set_error_literal (error, G_IO_ERROR, G_IO_ERROR_FAILED,
-                           "Not currently booted into an OSTree system and no OS specified");
-      goto out;
-    }
+    return glnx_throw (error, "Not currently booted into an OSTree system and no OS specified");
```

Comment:
Heh, I do also mostly work on my external (much larger) monitor, though still also work on my laptop fairly often which has a smaller resolution. I really like being able to have split buffers to look at two documents side by side, and when I do that, I get... (checking)... 94 char columns. So how about we say... 92 chars?